packages/CalendarCarousel/index.tsx

```diff
@@ -202,6 +226,35 @@ function CalendarCarousel<T>({
       numPrevMonthDays--;
     }
 
+    const markedDayEvents = [
+      {
+        selectedEventDate: new Date(2020, 7, 7),
+        events: 'walk dog',
+      },
+      {
+        selectedEventDate: new Date(2020, 7, 17),
+        events: 'birthday',
+      },
+      {
+        selectedEventDate: new Date(2020, 7, 27),
+        events: 'cooking',
+      },
+    ];
+
+    const [eventSwitch, setEventSwitch] = useState(0);
+    const dayEvents = [];
+    const markeddates = markedDayEvents.map((markeddates) => markeddates.selectedEventDate.getDate());
+    const markedmonths = markedDayEvents.map((markedmonths) => markedmonths.selectedEventDate.getMonth() - 1);
+    const isInside = (d: number) : boolean => {
+      if (markeddates.includes(d) && markedmonths.includes(month)) { return true; }
```

Comment:
This can be shortened to `markeddates.includes(d) && markedmonths.includes(month)`

Reply:
Thank you! I see 👍